dpdata/system.py

```diff
@@ -622,7 +623,7 @@ def from_deepmd_raw(self, folder, type_map = None) :
 
     @register_from_funcs.register_funcs("gro")
     @register_from_funcs.register_funcs("gromacs/gro")
-    def from_gromacs_gro(self, file_name) :
+    def from_gromacs_gro(self, file_name, format_atom_name=True) :
```

Comment:
Could you please explain in the doc string what is the behavior of `format_atom_name=True`?